packages/components/src/docs/Migration/ui-meta-info.mdx

```diff
@@ -0,0 +1,37 @@
+# Migration Guide: From `ui-meta-info` to `sd-meta`
+
+The new `sd-meta` is designed to replace the `ui-meta-info`. Rather than offering a complex component structure, they primarily focus on providing CSS styles for meta information.
```

Comment:
```suggestion
The new `sd-meta` is designed to replace the `ui-meta-info`. Rather than offering a complex component structure, they primarily focus on providing CSS styles for meta information. Make sure to provide correct semantics yourself, e. g. `<date>`.
```